### ISSUE TITLE
Improve Get-IdentitySource output

### ISSUE THREAD

**Issue by philiprusinov** [vmware/PowerCLI-Example-Scripts#459]
Hi Team,
Could you please improve a little bit Get-IdentitySource cmdlet from module VMware.VSphere.SSOAdmin.
I'm looking for some details regarding the certificates which are in use when LDAPS identity source is configured.

Now the output shows only the LDAP sources, without information when the certificates will expire. This is very useful information which is also unavailable on GUI since update from flash to HTML5.
 
```Get-IdentitySource -Server $VCSA | Where-Object -Property Name -eq mydomain.com | select *


Type                   : ActiveDirectory
Alias                  : mydomain
AuthenticationType     : password
AuthenticationUsername : mydomain\service-account
FriendlyName           : mydomain.com
PrimaryUrl             : ldaps://dc1.mydomain.com:3269
FailoverUrl            : ldaps://dc2.mydomain.com:3269
UserBaseDN             : DC=mydomain,DC=com
GroupBaseDN            : DC=mydomain,DC=com
Name                   : mydomain.com`
```

My proposal is to add certificate SN/ Validity date and etc.
BR,
Philip.

**Comment by dmilov** [comment]
Thanks for this request. I'll plan the fix for some of the next releases

**Comment by thinkdavid** [comment]
Also interested in this.

**Comment by dmilov** [comment]
Fixed in version 1.3.6
```powershell
> Get-IdentitySource -External

Type                   : ActiveDirectory
Alias                  :
AuthenticationType     : PASSWORD
AuthenticationUsername : administrator@sof-powercli-qe
FriendlyName           : sof-powercli-qe
PrimaryUrl             : ldap://sof-powercli-qe.eng.vmware.com:389
FailoverUrl            :
UserBaseDN             : CN=Users,DC=sof-powercli-qe,DC=eng,DC=vmware,DC=com
GroupBaseDN            : CN=Users,DC=sof-powercli-qe,DC=eng,DC=vmware,DC=com
Certificates           : {[Subject]
                           C=US, CN=10.23.80.118

                         [Issuer]
                           OU=VMware Engineering, O=photon-machine, S=California, C=US, DC=local, DC=vsphere, CN=CA

                         [Serial Number]
                           00F19BD28E35FB4F0E

                         [Not Before]
                           2/28/2020 10:47:51 AM

                         [Not After]
                           2/22/2030 10:47:50 AM

                         [Thumbprint]
                           71B11CA6E4861C86F74F33F802AA43F6C9E62F56
                         }
Name                   : sof-powercli-qe.eng.vmware.com
```

The new version is available on PSGallery

**Comment by philiprusinov** [comment]
Thanks Dimitar!
This is really great!
